config/routes.rb

```diff
@@ -8,7 +8,11 @@
   get "githook/master" => "githook#master"
 
   # Resources on the page
-  get '/vecktorn', to: redirect('http://old.fsektionen.se/vecktorn/signup.php'), as: :vecktorn_signup
+  get '/vecktorn', to: redirect('http://eepurl.com/bxH7lv'), as: :vecktorn_signup
+  get '/farad', to: redirect('http://www.farad.nu'), as: :farad
+  get '/facebook', to: redirect('https://www.facebook.com/Fsektionen'), as: :facebook
+  get '/twitter', to: redirect('https://www.twitter.com/Fsektionen'), as: :twitter
+  get '/youtube', to: redirect('https://www.youtube.com/user/fsektionen'), as: :youtube
```

Comment:
Please ensure these redirects are using code 301 (permanent) and not 302 (found).